tests/Integration/Behaviour/Features/Context/Domain/Product/UpdateStockFeatureContext.php

```diff
@@ -97,7 +128,20 @@ public function assertStockInformation(string $productReference, TableNode $tabl
         $this->assertStringProperty($productForEditing, $data, 'location');
         $this->assertIntegerProperty($productForEditing, $data, 'low_stock_threshold');
         $this->assertBoolProperty($productForEditing, $data, 'low_stock_alert');
-        $this->assertDateProperty($productForEditing, $data, 'available_date');
+
+        if (isset($data['available_date'])) {
```

Comment:
Maybe we could add this special check for null value inside `assertDateProperty` ? (maybe using an optional parameter to enable/disable this special check since not all DateTime comparison will need this)